lit/docs/pipelines/managing.lit

```diff
@@ -138,3 +141,22 @@
   order. If there are other pipelines that you haven't included in the command,
   they may appear in-between, before, or after the given set.
 }
+
+\section{
+    \title{\code{fly archive-pipeline}}{fly-archive-pipeline}
+
+    Pipelines can be archived via fly. This means that the existing pipeline will
+    be paused and stays hidden on the concourse UI. Archived pipelines retains their
```

Comment:
mostly grammar
```suggestion
    A pipeline can be archived via fly. This means that it will
    be paused and hidden in the web UI. Archived pipelines retain their
```